README.md

```diff
@@ -1,5 +1,4 @@
-
-⚡ **Serverless Inc. is hiring to build the next generation of serverless development tools, [join us!](https://www.serverless.com/company/jobs/)** 
+⚡ **Serverless Inc. is hiring to build the next generation of serverless development tools, [join us!](https://www.serverless.com/company/jobs/)**
```

Comment:
I think that change shouldn't be here - could you rebase on top of `master` and make sure that change is not a part of this PR?

Reply:
sorry about that, removed it